postreise/plot/plot_scatter_capacity_vs_curtailment.py

```diff
@@ -0,0 +1,126 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+
+from postreise.analyze.check import _check_resources_are_renewable_and_format
+from postreise.analyze.generation.capacity import get_capacity_by_resources
+from postreise.analyze.generation.curtailment import calculate_curtailment_time_series
+from postreise.analyze.helpers import get_plant_id_for_resources_in_area
+from postreise.analyze.time import change_time_zone, slice_time_series
+
+
+def plot_scatter_capacity_vs_curtailment(
+    scenario,
+    area,
+    resources,
+    time_zone="utc",
+    time_range=None,
+    area_type=None,
+    between_time=None,
+    dayofweek=None,
+    markersize=50,
+    fontsize=20,
+    title=None,
+    percentage=False,
+    show_plot=True,
+):
+    """Generate for a given scenario the scatter plot of the capacity (x-axis) vs
+        curtailment as a fraction of available resources (y-axis) of generators
+        located in area and fueled by resources over a time range.
```

Comment:
The last two lines should not be indented:
```
"""Generate for a given scenario the scatter plot of the capacity (x-axis) vs
curtailment as a fraction of available resources (y-axis) of generators located 
in area and fueled by resources over a time range.
```

Reply:
Done.